docs/book/src/user/quick-start.md

```diff
@@ -1456,6 +1456,29 @@ helm install --kubeconfig=./capi-quickstart.kubeconfig --repo https://raw.github
 
 For more information, see the [CAPZ book](https://capz.sigs.k8s.io/topics/addons.html).
 
+{{#/tab }}
+{{#tab OpenStack}}
+
+Before deploying the Calico CNI, configure the `cloud.conf` file for integration with your OpenStack environment:
+
+```bash
+cat > cloud.conf <<EOF
+[Global]
+auth-url=<your_auth_url>
+application-credential-id=<your_credential_id>
+application-credential-secret=<your_credential_secret>
+region=<your_region>
+domain-name=<your_domain_name>
+EOF
+```
+
+Next, create a Kubernetes secret using this configuration to securely store your cloud environment details.
+You can create this secret for example with:
+
+```bash
+kubectl create secret generic capi-quickstart-cloud-conf --from-file=cloud.conf
+```
+
```

Comment:
We should add here how to actually install it. Maybe something like this:

Now you are ready to deploy the external cloud provider!

```bash
kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/cloud-controller-manager-roles.yaml
kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/cloud-controller-manager-role-bindings.yaml
kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml
```

Alternatively, refer to the [helm chart](https://github.com/kubernetes/cloud-provider-openstack/tree/master/charts/openstack-cloud-controller-manager).